scale/job/data/job_data.py

```diff
@@ -295,26 +295,33 @@ def extend_interface_with_inputs_v5(self, interface, job_files):
         inputs = []
         input_files = deepcopy(interface.get_input_files())
         input_json = deepcopy(interface.get_input_json())
+        method = 'scale.job.data.job_data.JobData.extend_interface_with_inputs_v5'
+        logger.info("%s::Converting seed interface for job %s to v5", method, interface.get_name())
```

Comment:
We probably want both `name` and `jobVersion` here. As well as just debug level.